lib/camping/gear/kuddly.rb

```diff
@@ -0,0 +1,178 @@
+# frozen_string_literal: true
+# lib/camping/config
```

Comment:
What is this comment referencing?

Reply:
Old name for this file. I've now removed it.